src/app/store/chart/chart.selectors.ts

```diff
@@ -36,11 +37,11 @@ export const getStudentsFromClass = createSelector(
     return classes
       .filter((value) => regex.test(value.className))
       .reduce((acc, value: ClassModel) => {
-      acc.push({
-        data: [value.numOfStudents],
-        label: value.className
-      });
-      return acc;
-    }, []);
+        acc.push({
```

Comment:
I'm ok to leave it as is but also 
- you might add regex testing inside reduce and avoid one extra loop;
-  instead of push you can use `return [...acc, {data: [value.numOfStudents], label: value.className'}]`. It looks a bit shorter